src/bp/nlu-server/typings.ts

```diff
@@ -1,11 +1,49 @@
-import * as sdk from 'botpress/sdk'
-
 export interface TrainInput {
   language: string
-  topics: {
-    [topic: string]: sdk.NLU.IntentDefinition[] // TODO: change this for new API spec
-  }
-  entities: sdk.NLU.EntityDefinition[]
+  topics: Topic[]
+  enums: Enum[]
+  patterns: Pattern[]
+  complexes: Complex[]
   password: string
   seed?: number
 }
+
+export interface Topic {
+  name: string
+  intents: Intent[]
+}
+
+export interface Intent {
+  name: string
+  variables: Variable[]
+  examples: string[]
+}
+
+export interface Variable {
+  name: string
+  type: string
+}
+
+export interface Enum {
```

Comment:
isn't Enum a reserved keyword in ts , EnumEntity ?  Enumeration

Reply:
yeah, but it has to be in lowercase. I had few problems when naming a variable `enum` but this works fine. Do you want me to change it? `Enumeration` is fine to me.